doc/getting_started/_index.md

```diff
@@ -50,78 +46,88 @@ You can then **skip to step 4** (building software).
 
 ```console
-$ gcc --version
+python3 -m pip install --user -U pip setuptools
 ```
 
-If your version is lower, you have to upgrade it manually.
-For this, first, add `ubuntu-toolchain-r/test` PPA to your system using the following commands:
+The `pip` installation instructions use the `--user` flag to install without root permissions.
+Binaries are installed to `~/.local/bin`; check that this directory is listed in your `PATH` by running `which pip3`.
+It should show `~/.local/bin/pip3`.
+If it doesn't, add `~/.local/bin` to your `PATH`, e.g. by adding the following line to your `~/.bashrc` file:
 
-```console
-$ sudo apt install software-properties-common
-$ sudo add-apt-repository ppa:ubuntu-toolchain-r/test
+```console {title=~/.bashrc}
+export PATH=$PATH:~/.local/bin
 ```
 
-Next, install the necessary GCC and G++ version by:
+Now install additional Python dependencies:
 
 ```console
-$ sudo apt update
-$ sudo apt install gcc-9 g++-9
+cd $REPO_TOP
+pip3 install --user -r python-requirements.txt
 ```
 
-Finally, update the symbolic links for `gcc` and `g++` using these commands:
+### Adjust GCC version (if needed)
+
+On Ubuntu 18.04 the package `build-essential` includes the compilers `gcc-7` and `g++-7`.
+But for the OpenTitan project `gcc-9`/`g++-9` or higher is required, which has to be installed manually.
+Check that you have version 9 or higher of `gcc` installed by:
 
 ```console
-$ sudo ln -sf /usr/bin/gcc-9 /usr/bin/gcc
-$ sudo ln -sf /usr/bin/g++-9 /usr/bin/g++
+gcc --version
 ```
 
-Some tools in this repository are written in Python 3 and require Python dependencies to be installed through `pip`.
-We recommend installing the latest version of `pip` and `setuptools` (especially if on older systems such as Ubuntu 18.04) using:
+If your version is lower, you have to upgrade it manually.
+For this, first, add `ubuntu-toolchain-r/test` PPA to your system using the following commands:
 
 ```console
-$ python3 -m pip install --user -U pip setuptools
+sudo apt install software-properties-common
+sudo add-apt-repository ppa:ubuntu-toolchain-r/test
 ```
 
-The `pip` installation instructions use the `--user` flag to install without root permissions.
-Binaries are installed to `~/.local/bin`; check that this directory is listed in your `PATH` by running `which pip3`, which should show `~/.local/bin/pip3`.
-If it doesn't, add `~/.local/bin` to your `PATH`, e.g. by modifying your `~/.bashrc` file.
+Next, install the necessary GCC and G++ version by:
 
-Now install additional Python dependencies:
+```console
+sudo apt update
+sudo apt install gcc-9 g++-9
+```
+
+Finally, update the symbolic links for `gcc` and `g++` using these commands:
 
 ```console
-$ cd $REPO_TOP
-$ pip3 install --user -r python-requirements.txt
+sudo ln -sf /usr/bin/gcc-9 /usr/bin/gcc
+sudo ln -sf /usr/bin/g++-9 /usr/bin/g++
 ```
 
 ## Step 3: Install the LowRISC RISC-V Toolchain
```

Comment:
I think you can skip this step as Bazel will manage it all under the hood now, but I'm not confident that no one uses the toolchain outside of Bazel, or doesn't benefit from a locally installed copy.

Reply:
Are you referring to the toolchain installation or the GCC linking above, or both?

Reply:
Just the toolchain installation. Our unit tests use the host's GCC and we have features that depend on GCC-9 plus to compile without errors.

Reply:
I think its convenient to have a locally installed toolchain for debugging purposes, even though bazel should pick up the right version. Another option is to use [`update-alternatives`](http://charette.no-ip.com:81/programming/2011-12-24_GCCv47/) to manage the versions instead of forced symlinking, but I do like the simplicity of what you currently have, so I would probably just keep this approach.

Reply:
Alright, based on this discussion I think I'll leave the local installation instructions for now.